ckanext/s3filestore/tests/__init__.py

```diff
@@ -1,15 +1,19 @@
 # encoding: utf-8
 
-from ckantoolkit import config
-import boto3
+def _get_status_code(response):
+    """ Get the status code from a HTTP response.
+    Supports both Pylons/WebOb and Flask.
+    """
+    if hasattr(response, 'status_code'):
+        return response.status_code
+    elif hasattr(response, 'status_int'):
+        return response.status_int
+    else:
+        raise Exception("No status code found on %s" % response)
 
-# moto AWS mock is started externally on port 5000
-endpoint_url = config.get('ckanext.s3filestore.host_name', 'http://localhost:5000')
```

Comment:
so this is not needed?

Reply:
The S3 config should all be managed by `ckanext.s3filestore.uploader`. The tests make use of it at selected points.

This snippet was mostly for the benefit of the `setup_package` function, which nosetests used but pytest will not.